.yarn/patches/@metamask-assets-controllers-patch-74191f09e4.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/dist/NftController.js b/dist/NftController.js
+index ecbb338b3d52f0c9f40ccc81f5a50c8c41b542f0..eaeaa7df10d369cb01c03f16c061ade4e83968ec 100644
+--- a/dist/NftController.js
++++ b/dist/NftController.js
+@@ -778,6 +778,33 @@ class NftController extends base_controller_1.BaseControllerV1 {
+             }
+         });
+     }
++    /**
++     * Refetches NFT metadata and updates the state.
++     *
++     * @param nfts - Array of nfts
++     * @param networkClientId - The networkClientId that can be used to identify the network client to use for this request.
++     * @param userAddress - The current user address
++     */
++    updateNftMetadata(nfts, networkClientId, userAddress = this.config.selectedAddress) {
```

Comment:
Here are the steps how this file was generated:
1- Updated node-modules NftController.js (by adding my new fct)
2- `yarn patch @metamask/assets-controllers --update     ` 
3-  I updated the nftController.js file under `/private/var/folders/j2/z5v3y5wd23z_9ldp7tkknszh0000gn/T/xfs-6b6a5943/user `
4- yarn patch-commit -s /private/var/folders/j2/z5v3y5wd23z_9ldp7tkknszh0000gn/T/xfs-6b6a5943/user